pkgs/by-name/uw/uwsm/package.nix

```diff
@@ -66,7 +68,7 @@ stdenv.mkDerivation (finalAttrs: {
   postInstall =
     let
       wrapperArgs = ''
-        --prefix PATH : "${lib.makeBinPath finalAttrs.propagatedBuildInputs}"
+        --suffix PATH : "${lib.makeBinPath finalAttrs.propagatedBuildInputs}"
```

Comment:
Why are these deps propagatedBuildInputs?

Reply:
Only python deps need to be propagated and only for packages that other python packages might depend on (this is a leaf package w.r.t. python). There are some other cases where propagating an exact version of some package is important but these are just runtime inputs.